corehq/ex-submodules/pillowtop/feed/interface.py

```diff
@@ -60,9 +61,10 @@ def get_document(self):
         if not self.document and self.document_store and not self._document_checked:
             try:
                 self.document = self.document_store.get_document(self.id)
-            except DocumentNotFoundError:
+            except DocumentNotFoundError, e:
                 self.document = None
                 self._document_checked = True  # set this flag to avoid multiple redundant lookups
+                self.error_raised = e
```

Comment:
no, but it makes the calls order dependent which i think would be fairly confusing if anyone were to ever modify the code. bad to store the error raised?